<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Shumson" failures="1" tests="1" name="Default test(failed)" time="14.371" errors="0" timestamp="2023-12-09T11:53:22 EST">
  <testcase classname="reruntestcases.LoginTest" name="loginTest" time="14.299">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at reruntestcases.LoginTest.loginTest(LoginTest.java:20)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
</testsuite> <!-- Default test(failed) -->
